src/luxor_overrides.jl

```diff
@@ -273,3 +274,253 @@ function background(background_color)
     end
     Luxor.background(background_color)
 end
+
+"""
+    apply_transform(transform::Vector{Float64} , poly::Vector{Point})
+
+applies the transform , got by getmatrix() on every point in the poly
+and returns a new poly.
+
+move this to luxor_overrides_util.jl later.
+"""
+function apply_transform(transform::Vector{Float64}, poly::Vector{Point})
+    retpoly = Point[]
+    for pt in poly
+        res = cairotojuliamatrix(transform) * [pt[1], pt[2], 1]
+        push!(retpoly, Point(res[1], res[2]))
+    end
+    retpoly
+end
+
+"""
+updates CURRENT_JPATHS
+if used inside strokepath() pass action=:stroke
+and inside fillpath pass :fill.
+move this to luxor_overrides_util.jl later.
+"""
+function update_currentjpath(action::Symbol)
+    if CURRENT_FETCHPATH_STATE == true
+        #println("test strokepaths")
```

Comment:
Hi @ArbitRandomUser can't it be 
`if CURRENT_FETCHPATH_STATE == true` 
be changed to
`if CURRENT_FETCHPATH_STATE`? 
And similarly `if CURRENT_FETCHPATH_STATE == false` become `if !CURRENT_FETCHPATH_STATE` this is only if they can only be boolen but for 
CURRENT_FETCHPATH_STATE it should be the case?

Reply:
yes , i will change that .

Reply:
done , i pushed the changes , dont know why it doesn't show "outdated" above the review comment here